modules/cluster-logging-rn-5.3.11.adoc

```diff
@@ -0,0 +1,20 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-3-11"]
+= Logging 5.5.1
+This release includes link:https://access.redhat.com/errata/RHBA-[OpenShift Logging Bug Fix Release 5.3.11].
+
+[id="openshift-logging-5-3-11-bug-fixes"]
+== Bug fixes
+* Before this update, the Operator would not ensure that the pod was ready which caused the cluster to reach in an inoperable state during a cluster restart. With this update, the Operator will mark new pods as ready before continuing to a new pod during a restart which resolves the issue. (link:https://issues.redhat.com/browse/LOG-2871[LOG-2871])
```

Comment:
```suggestion
* Before this update, the Operator did not ensure that the pod was ready, which caused the cluster to reach an inoperable state during a cluster restart. With this update, the Operator marks new pods as ready before continuing to a new pod during a restart, which resolves the issue. (link:https://issues.redhat.com/browse/LOG-2871[LOG-2871])
```
Would it help to start that last sentence with: `This update resolves the issue: the Operator...`